ci_framework/hooks/playbooks/restart_nova_schedular.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Perform CRC login
+      ansible.builtin.include_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+
+    - name: Restart nova-scheduler to pick up cell1
+      ansible.builtin.command: >-
+        oc delete pod -n openstack -l service=nova-scheduler
```

Comment:
We are always assuming that we are under the same namespace but we are sometimes passing explicetly the namespace and other times not. In edpm_prepare I pass the namespace always for consistency, here do one of the options, always pass it like in edpm_prepare or assume that openstack (or the selected one, but I don't think we set openstack as default in any place, maybe we did it by selecting the oc project).